### ISSUE TITLE
JSON parsing

### ISSUE THREAD

**Issue by rupnikj** [qminer/qminer#451]
Fix how undefined is handled: `[undefined] -> [null], {a:undefined} -> {}`

**Comment by blazf** [comment]
What exactly does this issue refer to? Converting V8 Objects to PJsonVal?

**Comment by rupnikj** [comment]
Yes, its about convertin v8 objects (not strictly JSONs) to PJsonVal. If you send [undefined] you will not end up with [null] PJsonVal. 
It would be good if we would be consistent with: JSON.parse(JSON.stringify([undefined]))

**Comment by rupnikj** [comment]
Btw, both [undefined] and {a:undefined} are not JSON objects, because undefined is not a part of the standard, so it's not that our implementation is wrong...
